### DISCUSSION TITLE
Slow types and isolatedModules

### DISCUSSION THREAD

**Discussion by frangio** [jsr-io/jsr#464] [Q&A] [answered]
Is [`isolatedModules: true`](https://www.typescriptlang.org/tsconfig/#isolatedModules) sufficient to avoid having slow types in a project?

If not, is there a linter or some other tool to avoid accidentally shipping slow types?

**Comment by marvinhagemeister** [comment, marked as answer]
No, the slow types feature in JSR is basically the same as `isolatedDeclarations` in one of the upcoming TypeScript releases: https://github.com/microsoft/TypeScript/issues/47947
